src/logic.ts

```diff
@@ -99,15 +105,15 @@ export const getRequestFromNodeMessage = <T extends http.IncomingMessage>(
 
   const requestInit: fetch.RequestInit = {
     headers: headers,
-    method: request.method.toUpperCase(),
+    method: request.method!.toUpperCase(),
```

Comment:
do we want to fail when no method is present? why not fallback to `GET`?
as in ```request.method?.toUpperCase() || 'GET'```

Reply:
I thought the same, here is my reasoning: method is only missing when `request: node.IncomingMessage` is not produced by http or https servers because it is a reused type. (the only other option I know is by using the `request` package).  Falling back to GET hides an underlying configuration issue of the server and would assume "GET" as default method. I opted to have the code failing here.